pub/health_check.php

```diff
@@ -63,8 +63,10 @@
         }
         $cacheBackendClass = $cacheConfig[ConfigOptionsListConstants::CONFIG_PATH_BACKEND];
         try {
+            /** @var \Magento\Framework\App\Cache\Frontend\Factory $cacheFrontendFactory */
+            $cacheFrontendFactory = $objectManager->get(Magento\Framework\App\Cache\Frontend\Factory::class);
```

Comment:
Hi @andrewbess. 

I agree that the proposed change is "cleaner" and more maintainable. However, it violates backward compatibility. If an existing solution relies on `$cacheConfig[ConfigOptionsListConstants::CONFIG_PATH_BACKEND]` configuration, we will break the mentioned system by introducing this sort of change.

So, I would recommend do not refactor this part in the way as you did.